go.mod

```
@@ -3,54 +3,61 @@ module github.com/hashicorp/nomad-autoscaler
 go 1.19
```

```suggestion
go 1.20
```

We missed this one in https://github.com/hashicorp/nomad-autoscaler/pull/631, and all other plugins `go.mod` as well (we really need a script to handle these updates 😬 ).

---

Done!